config/default.js

```diff
@@ -70,5 +70,15 @@ module.exports = {
       host: 'api.geocode.earth',
       api_key: process.env.PELIAS_API_KEY || null
     }
+  },
+  geoip: {
+    host: 'api.ipstack.com/check',
+    api_key: process.env.GEOIP_API_KEY || null,
```

Comment:
I just realized that because API keys are provider-specific we should probably rename this to IPSTACK_API_KEY.